protocol/x/clob/ante/clob.go

```diff
@@ -53,17 +54,11 @@ func (cd ClobDecorator) AnteHandle(
 		return next(ctx, tx, simulate)
 	}
 
-	// Ensure that if this is a clob message then that there is only one.
-	// If it isn't a clob message then pass to the next AnteHandler.
-	isSingleClobMsgTx, err := IsSingleClobMsgTx(tx)
-	if err != nil {
+	// Check if the transaction is a valid clob tx
+	if err := IsValidClobMsgTx(tx); err != nil {
```

Comment:
`IsValidClobMsgTx` implies a boolean return value.

```suggestion
	if err := ValidateMsgTypeInClobTx(tx); err != nil {
```